web/src/components/series.astro

```diff
@@ -0,0 +1,462 @@
+---
+const series_100 = {
+  id: "100",
+  title: "FLARE 100: Core",
+  subseries_list: [
+    {
+      id: "101",
+      title: "101: Python APIs",
+      description: "Learn to use the Client API, Model Controller API, and Job API.",
+      cards: [
+        {
+          title: "Getting Started",
+          tags: ["beg.", "pytorch", "lightning", "sklearn", "tensorflow"],
+          description: "Getting started examples using the Client API, Model Controller API, and Job API for different frameworks.",
+          link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/getting_started"
+        },
+        {
+          title: "Client API",
+          tags: ["docs"],
+          description: "The FLARE Client API provides an easy way for users to convert their centralized, local training code into federated learning code.",
+          link: "https://nvflare.readthedocs.io/en/main/programming_guide/execution_api_type/client_api.html#client-api"
+        },
+        {
+          title: "Model Controller API",
+          tags: ["docs"],
+          description: "The FLARE ModelController API provides an easy way for users to write and customize FLModel-based controller workflows.",
+          link: "https://nvflare.readthedocs.io/en/main/programming_guide/controllers/model_controller.html#model-controller"
+        },
+        {
+          title: "Fed Job API",
+          tags: ["docs"],
+          description: "The FLARE FedJob API allows users to Pythonically define and create job configurations.",
+          link: "https://nvflare.readthedocs.io/en/main/programming_guide/fed_job_api.html",
+          colab_link: "https://colab.research.google.com/github/NVIDIA/NVFlare/blob/main/examples/tutorials/flare_api.ipynb"
```

Comment:
this has a colab link though, is it "docs"?

Reply:
good catch, will remove